oap-server/server-starter/src/main/java/org/apache/skywalking/oap/server/starter/OAPServerBootstrap.java

```diff
@@ -43,12 +42,10 @@ public static void start() {
             ApplicationConfiguration applicationConfiguration = configLoader.load();
             manager.init(applicationConfiguration);
 
-            manager.find(TelemetryModule.NAME)
+            manager.find(CoreModule.NAME)
                    .provider()
-                   .getService(MetricsCreator.class)
-                   .createGauge("uptime", "oap server start up time", MetricsTag.EMPTY_KEY, MetricsTag.EMPTY_VALUE)
-                   // Set uptime to second
-                   .setValue(System.currentTimeMillis() / 1000d);
+                   .getService(ServerStatusService.class)
+                   .bootedNow(System.currentTimeMillis());
```

Comment:
It doesn't mean the OAP starts up **successfully** when this line is executed, no?

Reply:
It is. I did this because, in https://github.com/apache/skywalking/issues/10051, you would see the booting uptime matter. So I moved this setup into the core as a unique mechanism.
I don't intend to replace the existing checker.